subprojects/docs/src/docs/userguide/compatibility.adoc

```diff
@@ -48,6 +48,27 @@ Gradle is tested with Kotlin 1.3.72 through 1.6.21.
 
 Gradle plugins written in Kotlin target Kotlin 1.4 for compatibility with Gradle and Kotlin DSL build scripts, even though the embedded Kotlin runtime is Kotlin 1.6.
 
+.Embedded Kotlin version
```

Comment:
I would put the Gradle version in the first column

Reply:
I did it like that to be aligned with the java table, but you are right, it makes more sense.